packages/story-editor/src/components/form/tags/input.js

```diff
@@ -55,9 +56,23 @@ const TextInput = styled(BaseInput).attrs({ type: 'text' })`
   margin: 3px 0;
 `;
 
+const SuggestionList = styled(List)`
+  display: block;
+  list-style-type: none;
+  width: 100%;
+  margin-top: 4px;
+  li {
+    padding: 4px;
+    width: 100%;
+    ${themeHelpers.focusableOutlineCSS()}
+    border-radius: ${({ theme }) => theme.borders.radius.small};
+  }
+`;
```

Comment:
Nice. I'm glad this simplified things. I think any time we can use less popups in this app, it's a win since there's so many in every interaction rn. 

My only design feedback would be to add a `border-top` with the same border as the `Border` component just so this component doesn't feel like it's floating in there

Reply:
Def, good note. Should be all fixed up.

<img width="317" alt="Screen Shot 2021-10-01 at 8 57 58 AM" src="https://user-images.githubusercontent.com/10720454/135651609-7abe6e66-4e9e-4aee-9e96-154073740a13.png">
<img width="297" alt="Screen Shot 2021-10-01 at 8 57 36 AM" src="https://user-images.githubusercontent.com/10720454/135651611-c79f6a9c-5549-493c-a1ed-884d6cc83eff.png">